transport-msg.tex

```diff
@@ -317,15 +307,15 @@ \subsubsection{Common Message Format}
           to a request (1).
       \item Bit[1]: Identifies if a message is a Transport Message (0) or a
           Bus Message (1).
-      \item Bit[2-7] Are reserverd for future use and must be zero.
+      \item Bit[2-7] Are reserved for future use and must be zero.
     \end{itemize}
   \item \field{msg_id}:
     Uniquely identifies which message definition applies (e.g., GET_DEVICES,
     GET_FEATURES, SET_CONFIG). The specific range or enumeration of types is
```

Comment:
Here GET_FEATURES must be fixed to GET_DEVICE_FEATURES (this is the case is a number of other places with the change to get/set features) so a global pass should be done on this.
Also the "get devices" being an optional bus message, we might want to gieve an other example like get_device_info